### PR TITLE
clients

### PR DESCRIPTION
**Pull request by maniksurtani**
Proposal for organising client libs.

cc @nerdrew

### CODE REVIEW

**Comment by maniksurtani** [comment]
Also cc @steved

**Comment by nerdrew** [comment]
LGTM

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# QuotaService client libraries
+
+## Protocol
+QuotaService listens over gRPC, using the protobuf service and messages defined in
+`../protos/quota_service.proto`. Additional endpoints (such as JSON over HTTP) may be defined as
```
Comment (nerdrew) [inline client/README.md]: Link to the github src?

**Comment by nerdrew** [comment]
Merge it

**Pull request merged**